include/rppt_tensor_filter_augmentations.h

```diff
@@ -93,6 +93,27 @@ RppStatus rppt_box_filter_gpu(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t
 RppStatus rppt_gaussian_filter_gpu(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32f *stdDevTensor, Rpp32u kernelSize, RpptROIPtr roiTensorPtrSrc, RpptRoiType roiType, rppHandle_t rppHandle);
 #endif // GPU_SUPPORT
 
+/*! \brief Sobel Filter augmentation on HOST backend for a NCHW layout tensor
```

Comment:
Initially planned to add support for only grey scale image
since edge detection on RGB images does not give meaningful output, so added NCHW here

Changed to NHWC/NCHW now, since we are converting to grey scale internally for RGB and using them